core/logbook/src/main/java/org/phoebus/logbook/LogEntrySubmissionResult.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019 European Spallation Source ERIC.
+ *
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; either version 2
+ *  of the License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ *
+ */
+
+package org.phoebus.logbook;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class LogEntrySubmissionResult {
+    private LogEntry logEntry;
+    private int httpStatus;
```

Comment:
`httpStatus`: We cannot assume that all logbook implementations are based on web services.
For example, the SNS logbook makes entries via direct RDB access, so there is no 'httpStatus'.
A generic `String error` would work for everybody, or just an `Exception` where whatever error you want to convey can be in the message. When using an `Exception`, the logbook submission can simply throw an exception, no need to _return_ an exception, though.